shiptendance/routes/index.js

```diff
@@ -0,0 +1,71 @@
+var express = require('express');
+var router = express.Router();
+var Pirate = require('../models/pirate');
+
+/* GET home page. */
+router.get('/', function(req, res, next){
+  // index view
+  Pirate.find({}, function(err, pirates){
+    if (err) console.log(err);
```

Comment:
You can pass this `err` to the `next` callback if you want to spit an error message to the page (i.e. `if (err) next(err);`. Otherwise, the error is happening but basically being ignored.